nym-vpn-app/src/main.tsx

```diff
@@ -83,6 +84,8 @@ dayjs.extend(duration);
     vpnMode: (await kvGet<VpnMode>('vpn-mode')) || defaultVpnMode,
     uiTheme: await getTheme(),
     welcomeChecked: (await kvGet<boolean>('welcome-screen-seen')) || false,
+    streamingOptimizedLabelSeen:
+      (await kvGet<boolean>(STREAMING_OPTIMIZED_LABEL_FEATURE_KEY)) || false,
```

Comment:
you don't need to use this constant for the key name, the key values are typed anyway (from tauri rust side)

also to keep consistency
```
await kvGet<boolean>('<key-name>'))
```